src/mlagility/analysis/analysis.py

```diff
@@ -169,19 +168,20 @@ def call_benchit(
         _store_traceback(model_info)
 
     else:
-        # Stats that we want to save into the model's state.yaml file
+        # Stats that we want to save into the build's mlagility_stats.yaml file
```

Comment:
The name `mlagility_stats` is not ideal since a hash is not really a stat. Maybe `mlagility_metadata` is preferable?

Reply:
Resolving this discussion to use this logic to gather initial set of results. 
We can address that suggestion as part of a future PR if we find it relevant.